src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-put.test.ts

```diff
@@ -72,7 +72,7 @@ describe("PUT /jira/connect/enterprise/app/:uuid", () => {
 
 	it("should use existing privateKey if new privateKey is not passed in as body", async () => {
 
-		let existingApp = await GitHubServerApp.install({
+		let existingApp: GitHubServerApp | null = await GitHubServerApp.install({
```

Comment:
Curious why it could be null now? Looking at the implementation of `GitHubServerApp::install`, looks like it always return a result, and I don't see that we've changed it somewhere.

Reply:
We call `findByPk` on line 107. Without passing null, we see the same error we do everywhere else we call this method

![Uploading Screenshot 2023-04-03 at 10.11.18 am.png…]()